types/password-hash-and-salt/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for password-hash-and-salt 0.1
+// Project: https://github.com/florianheinemann/password-hash-and-salt
+// Definitions by: Ali Taheri <https://github.com/alitaheri>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Password {
+    hash(cb: (error: string, hash: string) => void): void;
+    verifyAgainst(hash: string, cb: (error: string, verified: boolean) => void): void;
+}
+
+declare function password(password: string): Password;
+
+declare const _: typeof password;
```

Comment:
If the module uses `export = password` it's not technically correct to import with `import * as password` (one is a CommonJS export, the other is an ES6 import). There is a workaround though where adding `declare namespace password {}` (with no body) will allow that syntax.